src/scripts.js

```diff
@@ -38,17 +42,26 @@ showPantryRecipes.addEventListener("click", findCheckedPantryBoxes);
 searchForm.addEventListener("submit", pressEnterSearch);
 
 // GENERATE A USER ON LOAD
-function generateUser() {
-  user = new User(users[Math.floor(Math.random() * users.length)]);
-  let firstName = user.name.split(" ")[0];
-  let welcomeMsg = `
-    <div class="welcome-msg">
-      <h1>Welcome ${firstName}!</h1>
-    </div>`;
-  document.querySelector(".banner-image").insertAdjacentHTML("afterbegin",
-    welcomeMsg);
-  findPantryInfo();
-}
+// function generateUser() {
+//   user = new User(users[Math.floor(Math.random() * users.length)]);
+//   let firstName = user.name.split(" ")[0];
+//   let welcomeMsg = `
+//     <div class="welcome-msg">
+//       <h1>Welcome ${firstName}!</h1>
+//     </div>`;
+//   document.querySelector(".banner-image").insertAdjacentHTML("afterbegin",
+//     welcomeMsg);
+//   findPantryInfo();
+// }
+
```

Comment:
Looks nice, thanks for leaving that generateUser file just commented out so we can make sure we get the welcome message in there somehow.